sktime/forecasting/compose/_ensemble.py

```diff
@@ -316,14 +316,19 @@ def _aggregate(y, aggfunc, weights):
     return pd.Series(y_agg, index=y.index)
 
 
+def _weighted_gmean(a, axis=0, weights=None):
+    avg = np.exp(a)
+    return np.log(np.average(avg, axis=axis, weights=weights))
```

Comment:
just a question out of interest, why are we doing `exp` and `log`?

Reply:
@aiwalter I may be mis-remembering from when I did this in `performance_metrics` but I believe it is to avoid the cumulative product getting too large.

Reply:
yes, exactly - this has improved numerical behaviour (I think)

Reply:
but... it's the wrong way round (see below)